build-system/tasks/3p-vendor-helpers.js

```diff
@@ -42,7 +42,10 @@ async function buildVendorConfigs(options) {
     const watchFunc = () => {
       buildVendorConfigs(copyOptions);
     };
-    watch(SRCPATH).on('change', debounce(watchFunc, watchDebounceDelay));
+    watch(SRCPATH).on(
+      'change',
+      debounce(watchFunc, watchDebounceDelay, {leading: true, trailing: true})
```

Comment:
Remove the options object from here. (I assume type-checking caught this?)

Reply:
Type checking did catch it! The new build-system check is not part of my local testing...so I only find it out via CI. Will start type checking locally now though!

Reply:
Shoutout to @rileyajones. Build system type checking to the rescue!